website/content/docs/auth/jwt/oidc-providers/azuread.mdx

```diff
@@ -111,13 +111,22 @@ send `_claim_names` and `_claim_sources`. For example, returned claims might loo
 }
 ```
 
-The OIDC auth method role can be configured to include the user ID in the endpoint URL, 
-which will be used by Vault to retrieve the groups for the user:
-
-- In Azure, under the applications **API Permissions**, grant the following permissions:
-  - Microsoft Graph API permission [Directory.Read.All](https://docs.microsoft.com/en-us/graph/permissions-reference#application-permissions-19)
-
-- In Vault, set `"provider_config"` to Azure.
+The OIDC auth method role can be configured to include the user ID in the endpoint URL,
+which will be used by Vault to retrieve the groups for the user. Additional API permissions
+must be added to the Azure app in order to request the additional groups from the Microsoft
+Graph API.
+
+To set the proper permissions on the Azure app:
+
+1. Locate the application under "App Registrations" in Azure
```

Comment:
I think we generally prefer `1.` and autonumbering?

Reply:
Ah, yes. Done in [112caf1](https://github.com/hashicorp/vault/pull/18532/commits/112caf1e50fea159906a223d5548dda9490aab1e). Thanks!